baggers.js

```diff
@@ -1,5 +1,67 @@
 var data = {
   "speakers": [
+     {
+        "since": "2018-01-12",
+        "name": "Paul Pinault",
+        "bio": "Architecte Infrastructure, Intégration. Expert IoT, Ambassadeur Sigfox",
+        "picture": "https://www.isima.fr/wp-content/uploads/2014/02/Paul.jpg",
+        "websites": [
+          {
+            "name": "Blog, Disk91",
```

Comment:
This `name` is not allowed. See the list at https://github.com/brownbaglunch/bblfr_data#websites